openapi_handlers.go

```diff
@@ -10,15 +10,13 @@ import (
 //go:embed public
 var publicDocs embed.FS
 
-func PublicOpenApiv31(c echo.Context) error {
-	bytes, err := readOpenApiJson("public/openapi-3-v3.1.json")
-	if err != nil {
-		return err
-	}
-
-	return c.JSONBlob(http.StatusOK, bytes)
-}
+func PublicOpenApi(version string) echo.HandlerFunc {
+	return func(c echo.Context) error {
+		bytes, err := publicDocs.ReadFile("public/openapi-3-" + version + ".json")
```

Comment:
To throw in more context around this.  I believe that we should support REAL versioning. But for now we need to get the rewrite out.  

So I back this change for now ( as a crawl phase ) and would like to see real versioning handled in the future in follow up PRs.